flockcover/flock
# name of pdf renamed

**Website Deploy Checklist**

The website has 3 break points form media queries (desktop/tablet/mobile size).

* Test that all pages look okay on all page widths: 

/home
/payAsYouFly
/flyUnlimited
/textMeTheApp
/insurance/training
/insurance/commercial
/insurance/recreational
/enterprise
/about

* Test all CTA’s, buttons and links on each page

* Test that intercom pops up

* Test that the nav bar and links work correctly on both mobile AND desktop on both pages

* Check that FAQs collapse as they should

* Check that the ‘how Flock works’ slider component slides

* All of the above should be done on both desktop (chrome/safari/firefox) and mobile
